common/config.py

```diff
@@ -326,7 +329,10 @@ def __init__(self, config_path=None, data_path=None):
         self.inhibitCookie = None
         self.setupUdev = tools.SetupUdev()
 
-        tools.initiate_translation()
+        tools.initiate_translation(self.language())
+
+        # Workaround
+        self.default_profile_name = _('Main profile')
```

Comment:
Here the main profiles name is translated again now using the correct configured language. A bit dump I know. 😄